src/Submissions/Api/Areas/Admin/Controllers/RequestsController.cs

```diff
@@ -0,0 +1,399 @@
+using AutoMapper;
+using Biobanks.Data.Entities;
+using Biobanks.Shared.Services.Contracts;
+using Biobanks.Submissions.Api.Areas.Admin.Models;
+using Biobanks.Submissions.Api.Constants;
+using Biobanks.Submissions.Api.Models.Emails;
+using Biobanks.Submissions.Api.Services.Directory.Contracts;
+using Biobanks.Submissions.Api.Services.Directory.Dto;
+using Biobanks.Submissions.Api.Services.EmailServices.Contracts;
+using Biobanks.Submissions.Api.Utilities;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Biobanks.Submissions.Api.Areas.Admin.Controllers;
+public class RequestsController : Controller
+{
+  private INetworkService _networkService;
+  private IOrganisationDirectoryService _organisationService;
+  private readonly UserManager<ApplicationUser> _userManager;
+  private readonly ITokenLoggingService _tokenLog;
+  private readonly IEmailService _emailService;
+  private readonly IMapper _mapper;
+  public RequestsController(
+    INetworkService networkService,
+    IOrganisationDirectoryService organisationService,
+    UserManager<ApplicationUser> userManager,
+    ITokenLoggingService tokenLog,
+    IEmailService emailService,
+    IMapper mapper
+    )
+  {
+    _networkService = networkService;
+    _organisationService = organisationService;
+    _userManager = userManager;
+    _tokenLog = tokenLog;
+    _emailService = emailService;
+    _mapper = mapper;
+  }
+
+  public async Task<ActionResult> Requests()
```

Comment:
can't this just be `Index`, so the route is `/Requests` rather than `Requests/Requests`?
```suggestion
  public async Task<ActionResult> Index()
```

Reply:
View name might need changing too